### DISCUSSION TITLE
Spreading this project

### DISCUSSION THREAD

**Discussion by CleanMachine1** [Itai-Nelken/aptpac#8] [Ideas]
We should to spreading this project as it is a very simple but very elegant software.

We could look at AUR to put it within easy access of use.

We could also maybe look at the forum, and although the arch users maybe a little elitist, we could promote your idea there, however this should be done when that learning mode is done, otherwise we will get a lot of anger since arch users pride themselves with their precious pacman :rofl:

**Comment by Itai-Nelken** [comment]
good idea, I'll learn how to write a PKGBUILD for the AUR and create one once I finish the learning mode.

**Comment by Itai-Nelken** [comment]
@CleanMachine1 can you change the part about 'the better linux' in your review to 'the better linux in my opinion' please?

**Reply by CleanMachine1** [reply]
Sure, just change it
